packages/playwright-test/src/matchers/toMatchSnapshot.ts

```diff
@@ -292,6 +296,10 @@ export async function toHaveScreenshot(
   const testInfo = currentTestInfo();
   if (!testInfo)
     throw new Error(`toHaveScreenshot() must be called during the test`);
+
+  if (testInfo.config._ignoreSnapshots)
+    return { pass: !this.isNot, message: () => '' };
```

Comment:
I wonder if we could wait for a screenshot or two consecutive matching screenshots to better match real timings.

Reply:
I'd refrain from this for now.